packages/build-utils/test/unit.detect-workspace-managers.test.ts

```diff
@@ -0,0 +1,29 @@
+import path from 'path';
+import { detectFramework } from '../src/detect-framework';
+import workspaceManagers from '../src/workspaces/workspace-managers';
+import { FixtureFilesystem } from './utils/fixture-filesystem';
+
+describe('workspace-managers', () => {
+  describe.each([
+    ['npm', '21-npm-workspaces'],
+    ['pnpm', '23-pnpm-workspaces'],
+    ['yarn', '27-yarn-workspaces'],
+    [null, '22-pnpm'],
```

Comment:
Lets also test a fixture that has two lock files (I think @Ethan-Arrowood already added a fixture for this somewhere)

Reply:
Yes should be tests 25 and 26 if I recall.

Reply:
```
 PASS  test/unit.detect-workspace-managers.test.ts
  workspace-managers
    with detectFramework
      ✓ should detect a npm workspace for 21-npm-workspaces (2 ms)
      ✓ should detect a pnpm workspace for 23-pnpm-workspaces
      ✓ should detect a yarn workspace for 27-yarn-workspaces
      ✓ should detect a yarn workspace for 25-multiple-lock-files-yarn
      ✓ should detect a pnpm workspace for 26-multiple-lock-files-pnpm
      ✓ should not detect framework for 22-pnpm (1 ms)
```